cloud-controller-manager/cmd/digitalocean-webhook-server/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 DigitalOcean
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build the manager binary
+FROM golang:1.19 as builder
+
+WORKDIR /workspace
+
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+COPY vendor/ vendor/
+
+
+# Copy the go source
+COPY  cloud-controller-manager/cmd/digitalocean-webhook-server/main.go main.go
+COPY  cloud-controller-manager/do cloud-controller-manager/do
+RUN  go mod tidy
+RUN  go mod vendor
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -v -a -o doks_webhooks main.go
```

Comment:
what's `-v` for?

Reply:
verbosity

Reply:
It helps if a failing package is causing the image build to error